streamnative/README.md

```diff
@@ -0,0 +1,69 @@
+# StreamNative
+
+## Overview
+
+[StreamNative][1] provides an enterprise-grade messaging and event streaming platform built on Apache Pulsar. It offers scalable, real-time data streaming solutions with features like multi-tenancy, geo-replication, and seamless integration with cloud services.
+
+The StreamNative integration collects the below types of [metrics][2]:
```

Comment:
"collects the below" > Collects the following types of metrics

Reply:
Done.